### ISSUE TITLE
Help me, MariaDB Crashes and I dont know why.

### ISSUE THREAD

**Issue by Emporea** [MariaDB/mariadb-docker#307]
I use Nextcloud(18 currently). Everything was working fine until this day when i got this message when accessing my cloud via the webbrowser.

Internal Server Error  The server encountered an internal error  and was unable to complete your request. Please contact the server  administrator if this error reappears multiple times, please include the  technical details below in your report. More details can be found in  the server log.

I am using docker, or more specific docker compose for almost 2 years now. fpm alpine, splitted into 5 parts: mariadb, redis, app, web, and cron.
```yaml
version: '3'

services:
  nextcloud_db:
    image: mariadb
    command: --transaction-isolation=READ-COMMITTED --binlog-format=ROW
    restart: always
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - nextcloud_db:/var/lib/mysql
    environment:
      - MYSQL_ROOT_PASSWORD=
    env_file:
      - db.env

  nextcloud_redis:
    image: redis:alpine
    restart: always

  nextcloud_app:
    build: ./app
    restart: always
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - nextcloud:/var/www/html
    environment:
      - MYSQL_HOST=nextcloud_db
      - REDIS_HOST=nextcloud_redis
    env_file:
      - db.env
    depends_on:
      - nextcloud_db
      - nextcloud_redis

  nextcloud_web:
    build: ./web
    restart: always
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - nextcloud:/var/www/html:ro
    environment:
      - VIRTUAL_HOST=
      - LETSENCRYPT_HOST=
    depends_on:
      - nextcloud_app

  nextcloud_cron:
    image: nextcloud:fpm-alpine
    restart: always
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - nextcloud:/var/www/html
    entrypoint: /cron.sh
    depends_on:
      - nextcloud_db
      - nextcloud_redis

volumes:
  nextcloud_db:
  nextcloud:
```

The mariadb one has an error currently.

![image](https://user-images.githubusercontent.com/21969523/81991409-777ddf80-9641-11ea-824b-5101118d3975.png)

this is in the docker logs. this sounds way too complicated for me and i have no clue.

I have not made any changes to my system. I need my system working. It is in productive use.

let me know if you know anything related to this problem or if I should provide further data or what i should look for to get this fixed. Thank you.

**Comment by wglambert** [comment]
https://serverfault.com/a/705171
https://bugs.mysql.com/bug.php?id=83058

You could try backing up your data and mounting a [my.cnf](https://github.com/docker-library/docs/tree/master/mariadb#using-a-custom-mysql-configuration-file) with `innodb_force_recovery` of maybe 4? 
https://mariadb.com/kb/en/innodb-recovery-modes/
https://chepri.com/mysql-innodb-corruption-and-recovery/

I would also try asking over at the [Docker Community Forums](https://forums.docker.com/), [Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker). Since these repos aren't really a user-help forum

**Comment by sscherfke** [comment]
It looks like this issues was introduced with 10.4.13.  10.4.12 still works.

I started a docker container with the following command, connected to it and tried create some tables.  After this, I shut down the server:

```bash
IMAGE='mariadb:10.4.12'
# IMAGE='mariadb:10.4.13'
export MYSQL_PORT='33072'
docker run --rm \
        --env "MYSQL_DATABASE=test" \
        --env "MYSQL_ROOT_PASSWORD=rootmysql" \
        --name mariadb-test \
        --publish $MYSQL_PORT:3306 \
        $IMAGE \
        --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_520_ci \
        &> /tmp/mariadb.log &
```

Here is the container's output for both versions:

#### 10.4.12

```
2020-05-15 11:21:59+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 1:10.4.12+maria~bionic started.
2020-05-15 11:21:59+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2020-05-15 11:21:59+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 1:10.4.12+maria~bionic started.
2020-05-15 11:21:59+00:00 [Note] [Entrypoint]: Initializing database files


PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !
To do so, start the server, then issue the following commands:

'/usr/bin/mysqladmin' -u root password 'new-password'
'/usr/bin/mysqladmin' -u root -h  password 'new-password'

Alternatively you can run:
'/usr/bin/mysql_secure_installation'

which will also give you the option of removing the test
databases and anonymous user created by default.  This is
strongly recommended for production servers.

See the MariaDB Knowledgebase at http://mariadb.com/kb or the
MySQL manual for more instructions.

Please report any problems at http://mariadb.org/jira

The latest information about MariaDB is available at http://mariadb.org/.
You can find additional information about the MySQL part at:
http://dev.mysql.com
Consider joining MariaDB's strong and vibrant community:
https://mariadb.org/get-involved/

2020-05-15 11:22:02+00:00 [Note] [Entrypoint]: Database files initialized
2020-05-15 11:22:02+00:00 [Note] [Entrypoint]: Starting temporary server
2020-05-15 11:22:02+00:00 [Note] [Entrypoint]: Waiting for server startup
2020-05-15 11:22:02 0 [Note] mysqld (mysqld 10.4.12-MariaDB-1:10.4.12+maria~bionic) starting as process 123 ...
2020-05-15 11:22:02 0 [Note] InnoDB: Using Linux native AIO
2020-05-15 11:22:02 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2020-05-15 11:22:02 0 [Note] InnoDB: Uses event mutexes
2020-05-15 11:22:02 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2020-05-15 11:22:02 0 [Note] InnoDB: Number of pools: 1
2020-05-15 11:22:02 0 [Note] InnoDB: Using SSE2 crc32 instructions
2020-05-15 11:22:02 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2020-05-15 11:22:02 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
2020-05-15 11:22:02 0 [Note] InnoDB: Completed initialization of buffer pool
2020-05-15 11:22:02 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2020-05-15 11:22:02 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
2020-05-15 11:22:02 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2020-05-15 11:22:02 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2020-05-15 11:22:02 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2020-05-15 11:22:02 0 [Note] InnoDB: Waiting for purge to start
2020-05-15 11:22:02 0 [Note] InnoDB: 10.4.12 started; log sequence number 60972; transaction id 21
2020-05-15 11:22:02 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2020-05-15 11:22:02 0 [Note] Plugin 'FEEDBACK' is disabled.
2020-05-15 11:22:02 0 [Warning] 'user' entry 'root@70ba34846953' ignored in --skip-name-resolve mode.
2020-05-15 11:22:02 0 [Warning] 'user' entry '@70ba34846953' ignored in --skip-name-resolve mode.
2020-05-15 11:22:02 0 [Warning] 'proxies_priv' entry '@% root@70ba34846953' ignored in --skip-name-resolve mode.
2020-05-15 11:22:02 0 [Note] InnoDB: Buffer pool(s) load completed at 200515 11:22:02
2020-05-15 11:22:02 0 [Note] Reading of all Master_info entries succeeded
2020-05-15 11:22:02 0 [Note] Added new Master_info '' to hash table
2020-05-15 11:22:02 0 [Note] mysqld: ready for connections.
Version: '10.4.12-MariaDB-1:10.4.12+maria~bionic'  socket: '/var/run/mysqld/mysqld.sock'  port: 0  mariadb.org binary distribution
2020-05-15 11:22:03+00:00 [Note] [Entrypoint]: Temporary server started.
Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.
2020-05-15 11:22:36 10 [Warning] 'proxies_priv' entry '@% root@70ba34846953' ignored in --skip-name-resolve mode.
2020-05-15 11:22:36+00:00 [Note] [Entrypoint]: Creating database test

2020-05-15 11:22:36+00:00 [Note] [Entrypoint]: Stopping temporary server
2020-05-15 11:22:36 0 [Note] mysqld (initiated by: root[root] @ localhost []): Normal shutdown
2020-05-15 11:22:36 0 [Note] Event Scheduler: Purging the queue. 0 events
2020-05-15 11:22:36 0 [Note] InnoDB: FTS optimize thread exiting.
2020-05-15 11:22:36 0 [Note] InnoDB: Starting shutdown...
2020-05-15 11:22:36 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
2020-05-15 11:22:36 0 [Note] InnoDB: Buffer pool(s) dump completed at 200515 11:22:36
2020-05-15 11:22:38 0 [Note] InnoDB: Shutdown completed; log sequence number 60981; transaction id 24
2020-05-15 11:22:38 0 [Note] InnoDB: Removed temporary tablespace data file: "ibtmp1"
2020-05-15 11:22:38 0 [Note] mysqld: Shutdown complete

2020-05-15 11:22:38+00:00 [Note] [Entrypoint]: Temporary server stopped

2020-05-15 11:22:38+00:00 [Note] [Entrypoint]: MySQL init process done. Ready for start up.

2020-05-15 11:22:38 0 [Note] mysqld (mysqld 10.4.12-MariaDB-1:10.4.12+maria~bionic) starting as process 1 ...
2020-05-15 11:22:38 0 [Note] InnoDB: Using Linux native AIO
2020-05-15 11:22:38 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2020-05-15 11:22:38 0 [Note] InnoDB: Uses event mutexes
2020-05-15 11:22:38 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2020-05-15 11:22:38 0 [Note] InnoDB: Number of pools: 1
2020-05-15 11:22:38 0 [Note] InnoDB: Using SSE2 crc32 instructions
2020-05-15 11:22:38 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2020-05-15 11:22:38 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
2020-05-15 11:22:38 0 [Note] InnoDB: Completed initialization of buffer pool
2020-05-15 11:22:38 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2020-05-15 11:22:38 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
2020-05-15 11:22:38 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2020-05-15 11:22:38 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2020-05-15 11:22:38 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2020-05-15 11:22:38 0 [Note] InnoDB: Waiting for purge to start
2020-05-15 11:22:38 0 [Note] InnoDB: 10.4.12 started; log sequence number 60981; transaction id 21
2020-05-15 11:22:38 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2020-05-15 11:22:38 0 [Note] Plugin 'FEEDBACK' is disabled.
2020-05-15 11:22:38 0 [Note] Server socket created on IP: '::'.
2020-05-15 11:22:38 0 [Note] InnoDB: Buffer pool(s) load completed at 200515 11:22:38
2020-05-15 11:22:38 0 [Warning] 'proxies_priv' entry '@% root@70ba34846953' ignored in --skip-name-resolve mode.
2020-05-15 11:22:38 0 [Note] Reading of all Master_info entries succeeded
2020-05-15 11:22:38 0 [Note] Added new Master_info '' to hash table
2020-05-15 11:22:38 0 [Note] mysqld: ready for connections.
Version: '10.4.12-MariaDB-1:10.4.12+maria~bionic'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
2020-05-15 11:22:40 0 [Note] mysqld (initiated by: unknown): Normal shutdown
2020-05-15 11:22:40 0 [Note] Event Scheduler: Purging the queue. 0 events
2020-05-15 11:22:40 0 [Note] InnoDB: FTS optimize thread exiting.
2020-05-15 11:22:40 0 [Note] InnoDB: Starting shutdown...
2020-05-15 11:22:40 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
2020-05-15 11:22:40 0 [Note] InnoDB: Buffer pool(s) dump completed at 200515 11:22:40
2020-05-15 11:22:42 0 [Note] InnoDB: Shutdown completed; log sequence number 146503; transaction id 166
2020-05-15 11:22:42 0 [Note] InnoDB: Removed temporary tablespace data file: "ibtmp1"
2020-05-15 11:22:42 0 [Note] mysqld: Shutdown complete
```

#### 10.4.13

```
2020-05-15 11:23:18+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 1:10.4.13+maria~bionic started.
2020-05-15 11:23:18+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
2020-05-15 11:23:18+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 1:10.4.13+maria~bionic started.
2020-05-15 11:23:19+00:00 [Note] [Entrypoint]: Initializing database files


PLEASE REMEMBER TO SET A PASSWORD FOR THE MariaDB root USER !
To do so, start the server, then issue the following commands:

'/usr/bin/mysqladmin' -u root password 'new-password'
'/usr/bin/mysqladmin' -u root -h  password 'new-password'

Alternatively you can run:
'/usr/bin/mysql_secure_installation'

which will also give you the option of removing the test
databases and anonymous user created by default.  This is
strongly recommended for production servers.

See the MariaDB Knowledgebase at http://mariadb.com/kb or the
MySQL manual for more instructions.

Please report any problems at http://mariadb.org/jira

The latest information about MariaDB is available at http://mariadb.org/.
You can find additional information about the MySQL part at:
http://dev.mysql.com
Consider joining MariaDB's strong and vibrant community:
https://mariadb.org/get-involved/

2020-05-15 11:23:22+00:00 [Note] [Entrypoint]: Database files initialized
2020-05-15 11:23:22+00:00 [Note] [Entrypoint]: Starting temporary server
2020-05-15 11:23:22+00:00 [Note] [Entrypoint]: Waiting for server startup
2020-05-15 11:23:22 0 [Note] mysqld (mysqld 10.4.13-MariaDB-1:10.4.13+maria~bionic) starting as process 122 ...
2020-05-15 11:23:22 0 [Note] InnoDB: Using Linux native AIO
2020-05-15 11:23:22 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2020-05-15 11:23:22 0 [Note] InnoDB: Uses event mutexes
2020-05-15 11:23:22 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2020-05-15 11:23:22 0 [Note] InnoDB: Number of pools: 1
2020-05-15 11:23:22 0 [Note] InnoDB: Using SSE2 crc32 instructions
2020-05-15 11:23:22 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2020-05-15 11:23:22 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
2020-05-15 11:23:22 0 [Note] InnoDB: Completed initialization of buffer pool
2020-05-15 11:23:22 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2020-05-15 11:23:22 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
2020-05-15 11:23:22 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2020-05-15 11:23:22 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2020-05-15 11:23:22 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2020-05-15 11:23:22 0 [Note] InnoDB: Waiting for purge to start
2020-05-15 11:23:22 0 [Note] InnoDB: 10.4.13 started; log sequence number 60972; transaction id 21
2020-05-15 11:23:22 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2020-05-15 11:23:22 0 [Note] Plugin 'FEEDBACK' is disabled.
2020-05-15 11:23:22 0 [Warning] 'user' entry 'root@2ddd4058ba4d' ignored in --skip-name-resolve mode.
2020-05-15 11:23:22 0 [Warning] 'user' entry '@2ddd4058ba4d' ignored in --skip-name-resolve mode.
2020-05-15 11:23:22 0 [Warning] 'proxies_priv' entry '@% root@2ddd4058ba4d' ignored in --skip-name-resolve mode.
2020-05-15 11:23:22 0 [Note] InnoDB: Buffer pool(s) load completed at 200515 11:23:22
2020-05-15 11:23:22 0 [Note] Reading of all Master_info entries succeeded
2020-05-15 11:23:22 0 [Note] Added new Master_info '' to hash table
2020-05-15 11:23:22 0 [Note] mysqld: ready for connections.
Version: '10.4.13-MariaDB-1:10.4.13+maria~bionic'  socket: '/var/run/mysqld/mysqld.sock'  port: 0  mariadb.org binary distribution
2020-05-15 11:23:23+00:00 [Note] [Entrypoint]: Temporary server started.
Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.
2020-05-15 11:23:57 10 [Warning] 'proxies_priv' entry '@% root@2ddd4058ba4d' ignored in --skip-name-resolve mode.
2020-05-15 11:23:57+00:00 [Note] [Entrypoint]: Creating database test

2020-05-15 11:23:57+00:00 [Note] [Entrypoint]: Stopping temporary server
2020-05-15 11:23:57 0 [Note] mysqld (initiated by: root[root] @ localhost []): Normal shutdown
2020-05-15 11:23:57 0 [Note] Event Scheduler: Purging the queue. 0 events
2020-05-15 11:23:57 0 [Note] InnoDB: FTS optimize thread exiting.
2020-05-15 11:23:57 0 [Note] InnoDB: Starting shutdown...
2020-05-15 11:23:57 0 [Note] InnoDB: Dumping buffer pool(s) to /var/lib/mysql/ib_buffer_pool
2020-05-15 11:23:57 0 [Note] InnoDB: Buffer pool(s) dump completed at 200515 11:23:57
2020-05-15 11:23:59 0 [Note] InnoDB: Shutdown completed; log sequence number 60981; transaction id 24
2020-05-15 11:23:59 0 [Note] InnoDB: Removed temporary tablespace data file: "ibtmp1"
2020-05-15 11:23:59 0 [Note] mysqld: Shutdown complete

2020-05-15 11:23:59+00:00 [Note] [Entrypoint]: Temporary server stopped

2020-05-15 11:23:59+00:00 [Note] [Entrypoint]: MySQL init process done. Ready for start up.

2020-05-15 11:24:00 0 [Note] mysqld (mysqld 10.4.13-MariaDB-1:10.4.13+maria~bionic) starting as process 1 ...
2020-05-15 11:24:00 0 [Note] InnoDB: Using Linux native AIO
2020-05-15 11:24:00 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2020-05-15 11:24:00 0 [Note] InnoDB: Uses event mutexes
2020-05-15 11:24:00 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2020-05-15 11:24:00 0 [Note] InnoDB: Number of pools: 1
2020-05-15 11:24:00 0 [Note] InnoDB: Using SSE2 crc32 instructions
2020-05-15 11:24:00 0 [Note] mysqld: O_TMPFILE is not supported on /tmp (disabling future attempts)
2020-05-15 11:24:00 0 [Note] InnoDB: Initializing buffer pool, total size = 256M, instances = 1, chunk size = 128M
2020-05-15 11:24:00 0 [Note] InnoDB: Completed initialization of buffer pool
2020-05-15 11:24:00 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2020-05-15 11:24:00 0 [Note] InnoDB: 128 out of 128 rollback segments are active.
2020-05-15 11:24:00 0 [Note] InnoDB: Creating shared tablespace for temporary tables
2020-05-15 11:24:00 0 [Note] InnoDB: Setting file './ibtmp1' size to 12 MB. Physically writing the file full; Please wait ...
2020-05-15 11:24:00 0 [Note] InnoDB: File './ibtmp1' size is now 12 MB.
2020-05-15 11:24:00 0 [Note] InnoDB: Waiting for purge to start
2020-05-15 11:24:00 0 [Note] InnoDB: 10.4.13 started; log sequence number 60981; transaction id 21
2020-05-15 11:24:00 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
2020-05-15 11:24:00 0 [Note] Plugin 'FEEDBACK' is disabled.
2020-05-15 11:24:00 0 [Note] Server socket created on IP: '::'.
2020-05-15 11:24:00 0 [Warning] 'proxies_priv' entry '@% root@2ddd4058ba4d' ignored in --skip-name-resolve mode.
2020-05-15 11:24:00 0 [Note] InnoDB: Buffer pool(s) load completed at 200515 11:24:00
2020-05-15 11:24:00 0 [Note] Reading of all Master_info entries succeeded
2020-05-15 11:24:00 0 [Note] Added new Master_info '' to hash table
2020-05-15 11:24:00 0 [Note] mysqld: ready for connections.
Version: '10.4.13-MariaDB-1:10.4.13+maria~bionic'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
200515 11:24:01 [ERROR] mysqld got signal 11 ;
This could be because you hit a bug. It is also possible that this binary
or one of the libraries it was linked against is corrupt, improperly built,
or misconfigured. This error can also be caused by malfunctioning hardware.

To report this bug, see https://mariadb.com/kb/en/reporting-bugs

We will try our best to scrape up some info that will hopefully help
diagnose the problem, but since we have already crashed,
something is definitely wrong and this may fail.

Server version: 10.4.13-MariaDB-1:10.4.13+maria~bionic
key_buffer_size=134217728
read_buffer_size=2097152
max_used_connections=1
max_threads=102
thread_count=7
It is possible that mysqld could use up to
key_buffer_size + (read_buffer_size + sort_buffer_size)*max_threads = 760255 K  bytes of memory
Hope that's ok; if not, decrease some variables in the equation.

Thread pointer: 0x7fbf5c000c08
Attempting backtrace. You can use the following information to find out
where mysqld died. If you see no messages after this, something went
terribly wrong...
stack_bottom = 0x7fbfcc349dd8 thread_stack 0x49000
mysqld(my_print_stacktrace+0x2e)[0x55743082de8e]
mysqld(handle_fatal_signal+0x515)[0x5574302a9915]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x12890)[0x7fbfd67b5890]
mysqld(_ZN12Item_func_in7cleanupEv+0x37)[0x5574303d0267]
mysqld(_ZN11Query_arena10free_itemsEv+0x2d)[0x55743004fc2d]
mysqld(_ZN3THD19cleanup_after_queryEv+0x11b)[0x557430051b4b]
mysqld(_Z11mysql_parseP3THDPcjP12Parser_statebb+0x25b)[0x5574300a202b]
mysqld(_Z16dispatch_command19enum_server_commandP3THDPcjbb+0x16d5)[0x5574300a4705]
mysqld(_Z10do_commandP3THD+0x104)[0x5574300a5bb4]
mysqld(_Z24do_handle_one_connectionP7CONNECT+0x25e)[0x55743018223e]
mysqld(handle_one_connection+0x3d)[0x5574301822fd]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x76db)[0x7fbfd67aa6db]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x3f)[0x7fbfd51cc88f]

Trying to get some variables.
Some pointers may be invalid and cause the dump to abort.
Query (0x7fbf5c012010): ALTER TABLE ats_job ADD COLUMN send_report_qa BOOL NOT NULL
Connection ID (thread ID): 9
Status: NOT_KILLED

Optimizer switch: index_merge=on,index_merge_union=on,index_merge_sort_union=on,index_merge_intersection=on,index_merge_sort_intersection=off,engine_condition_pushdown=off,index_condition_pushdown=on,derived_merge=on,derived_with_keys=on,firstmatch=on,loosescan=on,materialization=on,in_to_exists=on,semijoin=on,partial_match_rowid_merge=on,partial_match_table_scan=on,subquery_cache=on,mrr=off,mrr_cost_based=off,mrr_sort_keys=off,outer_join_with_cache=on,semijoin_with_cache=on,join_cache_incremental=on,join_cache_hashed=on,join_cache_bka=on,optimize_join_buffer_size=on,table_elimination=on,extended_keys=on,exists_to_in=on,orderby_uses_equalities=on,condition_pushdown_for_derived=on,split_materialized=on,condition_pushdown_for_subquery=on,rowid_filter=on,condition_pushdown_from_having=on

The manual page at http://dev.mysql.com/doc/mysql/en/crashing.html contains
information that should help you find out what is causing the crash.
Writing a core file...
Working directory at /var/lib/mysql
Resource Limits:
Limit                     Soft Limit           Hard Limit           Units
Max cpu time              unlimited            unlimited            seconds
Max file size             unlimited            unlimited            bytes
Max data size             unlimited            unlimited            bytes
Max stack size            8388608              unlimited            bytes
Max core file size        unlimited            unlimited            bytes
Max resident set          unlimited            unlimited            bytes
Max processes             unlimited            unlimited            processes
Max open files            1048576              1048576              files
Max locked memory         65536                65536                bytes
Max address space         unlimited            unlimited            bytes
Max file locks            unlimited            unlimited            locks
Max pending signals       127654               127654               signals
Max msgqueue size         819200               819200               bytes
Max nice priority         0                    0
Max realtime priority     0                    0
Max realtime timeout      unlimited            unlimited            us
Core pattern: |/usr/lib/systemd/systemd-coredump %P %u %g %s %t %c ...
```

**Comment by Emporea** [comment]
I can not remember even updating the image. But i already started a restore from a day its been still working. If it continue to appear i might try to downgrade.

**Comment by Emporea** [comment]
Should I create a seperate bug report on https://mariadb.com/kb/en/reporting-bugs ?

**Comment by Emporea** [comment]
Downgrading to 10.4.12 is indeed a workaround. My Tables are not corrupted, this was my fear initially.

**Comment by grooverdan** [comment]
[MDEV-22563](https://jira.mariadb.org/browse/MDEV-22563) is the bug report associated with this issue and a fix will be available in 10.4.13 when it is released.

Feel free to use the https://jira.mariadb.org tracker for server problems that aren't container packaging issues.

**Comment by grooverdan** [comment]
typo in version above. 10.4.14 is now released and should fix your problems. Please report and issue if this or any other problem is still present.

**Comment by grooverdan** [comment]
MDEV-22563 fixed in 10.5.4, 10.4.14
